### DISCUSSION TITLE
"See API docs for all 180+ supported oAuth providers" is not clear enough

### DISCUSSION THREAD

**Discussion by ericblade** [feathersjs/feathers#3415] [General]
This might be that I'm not smart enough at 3am to figure out what this means, but...

![image](https://github.com/feathersjs/feathers/assets/1451847/f9746acb-c25a-4e6b-9190-1e67058b6230)

![image](https://github.com/feathersjs/feathers/assets/1451847/e7f4e3e4-d7a4-463c-9699-5b746df57c92)

this document is not particularly clear

**Comment by daffl** [comment]
In addition to the API docs, [the guide](https://feathersjs.com/guides/basics/login.html) has a complete example on how to set up an oAuth provider.
